examples/advanced_operations/add_app_campaign.py

```diff
@@ -0,0 +1,305 @@
+    # An App campaign cannot use a shared campaign budget.
+    # explicitly_shared must be set to false.
+    campaign_budget.explicitly_shared.value = False
+
+    # Retrieve the campaign budget service.
+    campaign_budget_service = client.get_service('CampaignBudgetService',
+                                                 version='v2')
+    # Submit the campaign budget operation to add the campaign budget.
+    response = campaign_budget_service.mutate_campaign_budgets(
+        customer_id, [campaign_budget_operation])
+    resource_name = response.results[0].resource_name
+    print(f'Created campaign budget with resource_name: {resource_name}')
+    return resource_name
+
+
+def _create_campaign(client, customer_id, budget_resource_name):
+    """Creates an app campaign under the given customer ID.
+
+    Args:
+        client: an initialized GoogleAdsClient instance.
+        customer_id: a client customer ID str.
+        budget_resource_name: the budget to associate with the campaign
+
+    Returns:
+        A resource_name str for the newly created app campaign.
+    """
+    campaign_service = client.get_service('CampaignService', version='v2')
+    campaign_operation = client.get_type('CampaignOperation', version='v2')
+    campaign = campaign_operation.create
+    campaign.name.value = f'Interplanetary Cruise App #{uuid4()}'
+    campaign.campaign_budget.value = budget_resource_name
+    # Recommendation: Set the campaign to PAUSED when creating it to 
+    # prevent the ads from immediately serving. Set to ENABLED once you've
+    # added targeting and the ads are ready to serve.
+    campaign.status = (client.get_type('CampaignStatusEnum', version='v2')
+        .PAUSED)
+    # All App campaigns have an advertising_channel_type of
+    # MULTI_CHANNEL to reflect the fact that ads from these campaigns are
+    # eligible to appear on multiple channels.
+    campaign.advertising_channel_type = client.get_type(
+        'AdvertisingChannelTypeEnum', version='v2').MULTI_CHANNEL
+    campaign.advertising_channel_sub_type = client.get_type(
+        'AdvertisingChannelSubTypeEnum', version='v2').APP_CAMPAIGN
+    # Define the bidding strategy during campaign creation. An App campaign
+    # cannot use a portfolio bidding strategy.
+    # App campaigns only support the TARGET_CPA bidding strategy.
+    campaign.bidding_strategy_type = (client
+        .get_type('BiddingStrategyTypeEnum', version='v2').TARGET_CPA)
+    # Set the target CPA to $1 / app install.
+    #
+    # campaign_bidding_strategy is a 'oneof' message so setting target_cpa
+    # is mutually exclusive with other bidding strategies such as
+    # manual_cpc, commission, maximize_conversions, etc.
+    # See https://developers.google.com/google-ads/api/reference/rpc
+    # under current version / resources / Campaign
+    campaign.target_cpa.target_cpa_micros.value = 1000000
+    # Set the App Campaign Settings. Other settings are on a per-ad basis.
```

Comment:
I think the sentence "Other settings are on a per-ad basis." requires the context that this is different from how it used to be. Either be more explicit about what those "other settings" are or remove.